compiler/pash_runtime.sh

```diff
@@ -153,6 +153,13 @@ else
         exit 1
     fi
 
+    # store functions for distributed execution
+    if [ "$distributed_exec" -eq 1 ]; then
+        declared_functions="${PASH_TMP_PREFIX}/pash_$RANDOM$RANDOM$RANDOM"
+        declare -f > $declared_functions
```

Comment:
Is this file just sourced by the worker that executes the graph?

Reply:
yeah, this file gets stored and the worker manager sends a copy for each worker with the subgraph.